draft-ietf-webpush-encryption.md

```diff
@@ -227,10 +227,15 @@ length indicates that the message has been truncated.
 
 # Mandatory Group and Public Key Format {#mti}
 
-User Agents that enforce encryption MUST expose an elliptic curve Diffie-Hellman
-share on the P-256 curve [FIPS186].  Public keys, such as are encoded into the
-"dh" parameter, MUST be in the form of an uncompressed point as described in
-[X.692].
+User Agents MUST expose an elliptic curve Diffie-Hellman share on the P-256
+curve [FIPS186].
+
+Public keys, such as are encoded into the "dh" parameter, MUST be in the form of
+an uncompressed point as described in [X.692] (that is, a 65 octet sequence that
+starts with a 0x04 octet).
```

Comment:
I like this clarification, it makes it easier to grasp for developers. I sent you a PR to the W3C API to make the same change there, given that most people won't bother looking X9.62 reference.

https://github.com/w3c/push-api/pull/171